llvm/lib/Analysis/AssumptionCache.cpp

```diff
@@ -43,8 +43,8 @@ static cl::opt<bool>
 
 SmallVector<AssumptionCache::ResultElem, 1> &
 AssumptionCache::getOrInsertAffectedValues(Value *V) {
-  // Try using find_as first to avoid creating extra value handles just for the
```

Comment:
I would not call this is a typo fix, more like a change of wording.
But even with that nitpick aside, I am not convinced that the new wording is any better. The original expresses the same idea with fewer characters.